### PR TITLE
Fix Bug 1175199, update webmaker styleguide page to reference new logo

### PR DESCRIPTION
**Pull request by schalkneethling**
@craigcook r? This is still blocked :( but at least the work is not lost yet.

### CODE REVIEW

**Comment by schalkneethling** [comment]
Some of the files have been added to the DAM. It does not contain all of the relevant file formats so, I asked Lizz to either upload the required formats or, we need to change the front end to just cater for downloading SVG and PNG formats.

Will update here once this has been resolved.

**Comment by schalkneethling** [comment]
Ready for liftoff Captain @craigcook :dizzy:

**Comment by schalkneethling** [comment]
@jpetto r?

File: bedrock/styleguide/templates/styleguide/identity/webmaker-branding.html [hunk 1]
```diff
@@ -27,23 +27,23 @@
     <li>use our logos to, or in connection with, content that disparages us or sullies our reputation</li>
   </ul>
   <div class="guidelines-logo">
-    <img src="{{ static('img/styleguide/identity/webmaker/logo.png') }}" alt="Logo" />
+    {{ high_res_img('styleguide/identity/webmaker/logo.png', {'alt': 'Logo'}) }}
```
Comment (jpetto) [inline bedrock/styleguide/templates/styleguide/identity/webmaker-branding.html]: Need a `height` and `width` set so the high res version is constrained to the regular res dimensions.

**Comment by jpetto** [comment]
Just needs `height` and `width` attributes set on high res images. Otherwise, looking good. r+wc

**Comment by schalkneethling** [comment]
@jpetto complete oversight, my bad. Updated.

**Comment by jpetto** [comment]
Looks good! :left_right_arrow:

**Pull request merged**